torch/_lobpcg.py

```diff
@@ -11,7 +11,98 @@
 from .overrides import has_torch_function, handle_torch_function
 
 
-__all__ = ['lobpcg']
+__all__ = ['lobpcg', 'lobpcg2']
+
+class LOBPCGAutogradFunction(torch.autograd.Function):
+
+    @staticmethod
+    def forward(ctx,
+                A,              # type: Tensor
+                k,              # type: Optional[int]
+                B,              # type: Optional[Tensor]
+                X,              # type: Optional[Tensor]
+                n,              # type: Optional[int]
+                iK,             # type: Optional[Tensor]
+                niter,          # type: Optional[int]
+                tol,            # type: Optional[float]
+                largest,        # type: Optional[bool]
+                method,         # type: Optional[str]
+                tracker,        # type: Optional[None]
+                ortho_iparams,  # type: Optional[Dict[str, int]]
+                ortho_fparams,  # type: Optional[Dict[str, float]]
+                ortho_bparams,  # type: Optional[Dict[str, bool]]
+                ):
+        # type: (...) -> Tuple[Tensor, Tensor]
+
+        D, U = lobpcg(A, k, B, X, n, iK, niter, tol, largest, method, tracker, ortho_iparams, ortho_fparams, ortho_bparams)
```

Comment:
```suggestion
        D, U = torch.symeig(A, eigenvectors=True, upper=True)
        D = D[..., -k:]
        U = U[..., :, -k:]
```
@pearu, this fails the gradcheck, so, I guess, it is the issue with backward. Hmm...